hummingbot/templates/conf_fee_overrides_TEMPLATE.yml

```diff
@@ -20,3 +20,225 @@ binance_percent_fee_token:  # BNB
 binance_maker_percent_fee:  # 0.75
 binance_taker_percent_fee:  # 0.75
 binance_buy_percent_fee_deducted_from_returns:  # True
+
+# List of supported Exchanges for which the user's conf/conf_fee_override.yml
+# will work. This file currently needs to be in sync with hummingbot list of
+# supported exchanges
+altmarkets_buy_percent_fee_deducted_from_returns:
```

Comment:
Please remember to update the `template_version` when changing a `*TEMPLATE.yml` file